config/routes.rb

```diff
@@ -6,7 +6,7 @@
   get '/prizes', to: 'prizes#index'
   post '/prizes', to: 'prizes#create'
   get '/socials', to: 'socials#show'
-  post '/win-prize', to: 'prizes#win_prize'
+  post '/prize', to: 'prizes#win_prize'
```

Comment:
@zediogoviana Any restful / Rails conventions violated here?

Reply:
What does this `win_prize` do? Either way is not restful because it should only be only `CRUD` operations... I believe this would have to be namespace into `leads` and do something like `post '/leads/:lead/prize', to: lead_prizes#create` to be restful, because I'd be creating a `prize` for that specific `lead`. 
But this would require more changes than just updating the route.

Reply:
Link to [Rails docs](https://guides.rubyonrails.org/routing.html#nested-resources)

Reply:
Got it! Will leave this change for now as is.

If you are curious about making it restful, perhaps you can start a thread in Slack. @beasrodrigues24 @RicardoVaz1 @jreis13 (This would be backend refactoring)

Reply:
Thank you @zediogoviana 💙